node/src/main/scala/com/wavesplatform/database/RocksDBWriter.scala

```diff
@@ -744,13 +744,16 @@ class RocksDBWriter(
       }
 
       this.generationPeriodOf(h).foreach { currPeriod => // None checked in Caches
```

Comment:
Here we checking that feature is activated. There is no "generation period" before the feature activation. See also `GenerationPeriod.from`

Reply:
Thank you, fixed